apps/frontend/vite.config.ts

```diff
@@ -13,12 +13,7 @@ export default defineConfig({
 			project: './project.inlang',
 			outdir: './src/lib/paraglide'
 		}),
-		sentrySvelteKit({
-			sourceMapsUploadOptions: {
-				org: 'synergyai',
-				project: 'sveltekit-langgraph'
-			}
-		})
+		sentrySvelteKit()
```

Comment:
Why did you chose to remove this rather than make org and project configurable? 
Having source maps is really a HUGE help if we ever deal with frontend bugs!